src/firefly/js/tables/TablesCntlr.js

```diff
@@ -357,7 +357,7 @@ function tblResultsAdded(action) {
             var {tbl_id, title, options={}, tbl_ui_id} = action.payload;
 
             title = title || tbl_id;
-            options = Object.assign({tbl_group: 'main', removable: true}, options);
+            options = Object.assign({tbl_group: 'main', removable: true, setAsActive:true}, options);
```

Comment:
That would mean that in order to activate a particular table, i need to pass on the attribute 'setAsActive:false' to any other table request?

Reply:
I would not put it that way, because there's an action to set a table active.

This added option is useful in the case when you know whether or not this table should be active by default when added.